### DISCUSSION TITLE
Customizing mouse event behaviors

### DISCUSSION THREAD

**Discussion by dhrblav** [molstar/molstar#465] [Q&A] [answered]
Is it possible to customize some default mouse event behaviors?
For example I would like to prevent camera reset when clicking in void space, or prevent the default representation of surroundings when clicking on an amino acid.
Thank you.

**Comment by dsehnal** [comment, marked as answer]
Yeah, you can do that. In some instances it's as simple as changing a param or changing the plugin spec. Sometimes you will have to write your own ``PluginBehavior`` with the desired functionality.

Good place to start is here https://github.com/molstar/molstar/blob/master/src/mol-plugin/spec.ts#L120.
